tests/test_interfaces.py

```diff
@@ -32,6 +37,18 @@
 from nwb_conversion_tools.utils.conversion_tools import get_default_nwbfile_metadata
 
 
+class test(TestCase):
+    def test_import_assertions(self):
+        if platform == "darwin" and version.parse(python_version()) < version.parse("3.8"):
```

Comment:
@CodyCBakerPhD to be sure, the tests are not running for MacOS so far, right?
Is this test case ever running? I couldn't find the logs for it in the tests and the if/else seems to raise an error or skip it with a message

Reply:
It should be on this branch; this branch runs only 3.7 so should skip on mac but pass on windows full and linux full, even without GIN data. I'll see if I can find it in CI logs